src/lib/__tests__/insights-listener-test.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import withInsightsListener from '../insights/listener';
+import { mount } from 'enzyme';
+
+describe('withInsightsListener', () => {
+  const setup = Hits => {
```

Comment:
`Hits` is not typed at this point.

Reply:
Hits is just a mock for functional component
As you know they are rather hard to type and we keep limited typing in tests.

Reply:
changed to `React.ComponentType<any>`